docs/team/zichen-3974.md

```diff
@@ -3,35 +3,49 @@ layout: page
 title: Zichen's Project Portfolio Page
 ---
 
-### Project: eduMate
+### Project: EduMate
 
-eduMate is a desktop app designed for NUS students to manage their academic and social lives by lowering the barriers to meet up and make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+EduMate is a desktop app designed for NUS students to manage their academic and social lives by lowering the barriers to meet up and make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **User Profile Window**
-  * What it does: allows the user to view their personal information in a separate window.
-  * Justification: To be added.
-  * Highlights: To be added.
+* **New Feature:** Added the ability to organise/unorganise a meet up. (Pull requests [#228](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/228), [#249](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/249) )
+  * What it does: allows the user to organise a customised meeting or choose a meet up recommendation generated by the `Meet` command. The user can remove any organised meetings with the `Unorganise` command.
+  * Justification: This feature allows users to track and manage meet ups to keep their schedule organised.
+  * Highlights: This enhancement affects existing commands such as `Meet`, `Edit`, `Delete` and hence required additions to those commands.
 
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s2.github.io/tp-dashboard/?search=zichen-3974&breakdown=true)
+* **Enhancements to existing features**:
+  * Refactor UI (Pull requests [#58](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/58), [#78](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/78), [#178](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/178), [#228](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/228), [#247](https://github.com/AY2223S2-CS2103T-W14-2/tp/pull/247))
+    * Restructure UI to include new components such as the scheduled meet up pane, the meet recommendation pane, the user profile pane and the sidebar. 
```

Comment:
These could potentially have links to the file on GitHub; they are actually quite significant (and a lot of work too).

Reply:
Hmm makes sense, will add some. Thanks!